index.html

```diff
@@ -1,26 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,600,800' rel='stylesheet' type='text/css'>
     	<meta charset="utf-8">
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Relaxr</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
+	
 	<body>
-		<div id="container">
+		<div class="container" id="container">
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+				<h1>Relaxr</h1>
+			
+				<h2>Get piece of mind with a single tap</h2>
+
+				<div class="btn">
+					<a href="#">Get it Now</a>
+				</div>
+
 			</header>
+			
+			<div class="blocktwo">
```

Comment:
@humphriescarly Nice way to break up the page into manageable portions! (blocktwo, blockthree, blockfour)